client/src/routes/challs.js

```diff
@@ -212,6 +250,25 @@ const Challenges = ({ classes }) => {
             }
           </div>
         </div>
+        <div class={`frame ${classes.frame}`}>
+          <div class='frame__body'>
+            <div class='frame__title title'>Tags</div>
+            {
+              Object.keys(tags).sort((a, b) => a.localeCompare(b)).map((metatag) => {
+                return (<div><h5 class='frame__title title'>{metatag}</h5>
+                  {Object.entries(tags[metatag]).sort((a, b) => a[0].localeCompare(b[0])).map(([tag, checked]) => {
+                    return (
+                      <div key={tag} class='form-ext-control form-ext-checkbox'>
+                        <input id={`tag-${tag}`} data-tag={tag} data-metatag={metatag} class='form-ext-input' type='checkbox' checked={checked} onChange={handleTagsCheckedChange} />
```

Comment:
This will have an `id` conflict if two metatags have the same tag (which I don't think is prevented anywhere else)